### PR TITLE
Group 1 add the long passage translation and the short passage translation with the first paragraph

### PR DESCRIPTION
**Pull request by zl117**
Comments and thoughts:
    From what I see, Alibaba is really dedicating itself to make this world a better place, to solve the current problem including the zombie process, they made a bold move where they create different mode for the init processes. The greatest part is that since this project is open source right now, people from all over the world could contribute to this project and benefit from this. As a large company, what they desire is that they could use their experience to help those small business to make this transition from the bare metal or vms to containers.
    I think we could learn from what engineers have been doing for centuries. Since there is always trade offs in an engineering problem, like we trade space for time, trade safety for speed. In this case, could we reduce the granularity, to divide the user cases into smaller cases. Providing start up options for the users to give them a choice based on their requirements?

### CODE REVIEW

**Comment by zl117** [comment]
this is for #24

File: blog-cn/Group1:Deep analysis of PouchContainer technology.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Deep analysis of PouchContainer's rich container technology
+
+PouchContainer is an efficient, open-source, lightweight Enterprise Rich Container Engine Technology developed by the Alibaba group. This technology presents the characteristics of strong isolation, high portability, and low resource consumption for a container. With the help of PouchContainer, corporations can rapidly deploy their business into containers and improve the utilization of physical resources in their data centers under hyperscale at the same time.
+
+PouchContainer originates from the thoughts of Alibaba's engineers to the problems they had in their business scenarios. These engineers have dedicated enormous time and efforts in delivering internet applications, which could be proved by the characteristics of the PouchContainer listed above. Thanks to the mass scale of Alibaba, this technology embraced its first opportunity to be fully tested on its ability to support business at Nov 11th after it comes to the world.  After declared as an open source project, this new technology is positioned to serve for the public and to help enterprises quickly realize the containerization of their business.
```
Comment (RuiteGuo) [inline blog-cn/Group1:Deep analysis of PouchContainer technology.md]: after it came to the world

File: blog-cn/Group1:Deep analysis of PouchContainer technology.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+![image1.png | center | 180x60](https://cdn.yuque.com/lark/0/2018/png/65333/1526059437042-3ae3f269-7795-44b2-a618-e6af68dc9f21.png "")
+
+When it first comes to container technology, Alibaba had an amazing amount of backlogs inside at that time. How to quickly containerize these backlogs is the key problem of this technology when it was first promoted inside the company. Nowadays, the open source container technology is becoming more and more popular. Many companies may still be worrying about how to deal with the same problem.  In the cloud native field, many of the advanced concepts promoted by the CNCF Foundation are based on business containerization. If enterprises fall behind this trend, they will be overwhelmed by the follow-up problems brought by container management, let alone enjoying the benefits of open source projects like Service Mesh.
```
Comment (JeromeTan1997) [inline blog-cn/Group1:Deep analysis of PouchContainer technology.md]: According to Oxford dictionary, backlog means "An accumulation of uncompleted work or matters needing to be dealt with". It is not accurate to describe the existing businesses Alibaba has. A better translation could be "legacy businesses"

File: blog-cn/Group1:Deep analysis of PouchContainer technology.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+PouchContainer's rich container model, having taken industry's large number of applications and system service delivery applications into consideration, results in the design of init process inside the rich container armed with the ability to manage multiple system service processes.
```
Comment (RuiteGuo) [inline blog-cn/Group1:Deep analysis of PouchContainer technology.md]: resulting in

File: blog-cn/Group1:Deep analysis of PouchContainer technology.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+PouchContainer's rich container mode allows users to easily specify the application's start and stop execution hooks: prestart hook and poststop hook. The operation and maintenance team specifies the prestart hook, which can help the application to perform some initialization operations in the container that meet the operation and maintenance requirements before running, such as: initializing the network routing table, obtaining the application execution permission, and downloading the certificate required for the runtime. The same team specifies the poststop hook, which can help the application to perform unified after-work management after the end of the operation or abnormal exit. For example, the intermediate data is cleaned up for the pure environment at the next startup; if it is abnormally exited, the error can be reported immediately which is able to meet operational requirements.
```
Comment (RuiteGuo) [inline blog-cn/Group1:Deep analysis of PouchContainer technology.md]: 从句不能修饰整个句子， 不能用which 指代整句 “the error can be reported immediately”

**Comment by Starnop** [comment]
Thanks for your contribution. :)

**Pull request closed without merging**